### DISCUSSION TITLE
Github copilot  could not connect to server.extenstion activation failed

### DISCUSSION THREAD

**Discussion by Muhammad-Atif-K** [AammarTufail/python_ka_chilla2023#31] [Q&A]
<img width="956" alt="Screenshot 2023-01-07 124529" src="https://user-images.githubusercontent.com/121885523/211140140-e5a5859a-c838-4a9e-a623-d245221ed77c.png">

How to solve this error("Github copilot  could not connect to server.extenstion activation failed")
screen shot attached

**Comment by SahilJatoi744** [comment]
Apka github copilot activated nh usse activate kary

**Reply by Muhammad-Atif-K** [reply]
kaha se activate hoga?
